typescript/cli/src/context/strategies/chain/ChainResolverFactory.ts

```diff
@@ -0,0 +1,46 @@
+import { MultiChainResolver } from './MultiChainResolver.js';
+import { SingleChainResolver } from './SingleChainResolver.js';
+import { ChainResolver } from './types.js';
+
+enum CommandType {
```

Comment:
The important map in this file is buried too deep. It was cause confusion in the future as junior devs try to add commands but don't know about this obscure requirement. 

A better solution would be to have this defined where the commands themselves get defined. Better cohesion. But since you want this information in `context.ts`, that might not be possible without creating import cycles that will bite us later. I'm not 100% sure.

If we do definitely need a manually curated map, then combine it with `signCommands.ts` and put it in `/src/commands` so there's only one extra touch-point when creating new commands